org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/views/RuleDescriptionWebView.java

```diff
@@ -26,23 +26,35 @@
 import org.eclipse.ui.IWorkbenchPart;
 import org.sonarlint.eclipse.core.internal.SonarLintCorePlugin;
 import org.sonarlint.eclipse.core.internal.markers.MarkerUtils;
-import org.sonarlint.eclipse.core.internal.resources.SonarLintProject;
 import org.sonarlint.eclipse.ui.internal.SonarLintUiPlugin;
-import org.sonarlint.eclipse.ui.internal.SonarQubeUrls;
 
 /**
  * Display details of a rule in a web browser
  */
-public class RuleDescriptionWebView extends AbstractLinkedSonarWebView<IMarker>implements ISelectionListener {
+public class RuleDescriptionWebView extends AbstractLinkedSonarWebView<IMarker> implements ISelectionListener {
 
   public static final String ID = SonarLintUiPlugin.PLUGIN_ID + ".views.RuleDescriptionWebView";
 
+  private static final String CSS = "<style type=\"text/css\">body {    font-family: Helvetica Neue,Segoe UI,Helvetica,Arial,sans-serif;    font-size: 13px;    line-height: 1.23076923;}\n"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 188 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)